packages/wrangler/src/__tests__/helpers/run-wrangler.ts

```diff
@@ -8,7 +8,8 @@ export async function runWrangler(cmd?: string) {
   try {
     await main(cmd?.split(" ") ?? []);
   } catch (e) {
-    e.message = normalizeSlashes(stripTimings(e.message));
-    throw e;
+    const err = e as Error;
```

Comment:
typescript refuses to let me annotate e as an Error, but then the recommended fix is to cast e as an Error 🤷‍♂️

Reply:
The reason is that we cannot guarantee that `e` is an `Error`, since it is possible to throw `any`thing[sic], e.g. `throw "some string";`.

The most watertight solution is to use a type guard to narrow the caught value:

```
} catch(e) {
  if (e instanceof Error) {
    // do stuff with e as an Error.
  } else {
    throw e;
  }
}
```

Reply:
Noted, did this in a few places as seemed appropriate.